.textlintrc

```diff
@@ -185,17 +185,9 @@
       "weasel": false,
       "so": false,
       "thereIs" :false
-    },
-    "no-dead-link": {
```

Comment:
Why is this removed?

Reply:
I understand this is the explanation: https://github.com/telefonicaid/iotagent-json/pull/515#issuecomment-736355544

Makes sense. NTC